submissions/description/server_security_misconfiguration/insecure_ssl/template.md

```diff
@@ -0,0 +1,23 @@
+# Insecure SSL
+
+## Overview of the Vulnerability
+
+Insecure SSL refers to implementation flaws within the configuration of Secure Socket Layer (SSL)/Transport Layer Security (TLS), the security of the transport layer through encryption.
+
+The insecure configuration of  SSL within this application can lead to the connection between client and server being vulnerable. An attacker can use this weakness to execute a Denial of Service (DoS) attack, or Person-in-The-Middle (PiTM) the connection between the client and server to view or manipulate data in transit.
+
+## Business Impact
+
+Insecure SSL can lead to reputational damage for the business due to a loss in confidence and trust by users who can see that the transport layer security is insecure. 
+
+## Steps to Reproduce
+
+1. Use a tool such as SSLScan, TestSLL, or SSLyze to find that the application is using the following insecure SSL:
```

Comment:
The wording feels off here, try something like

Use a tool such as SSLScan, TestSLL, or SSLyze and the results show that the application is using the following Insecure SSL: